pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -115,7 +115,8 @@ func (pl *TaintToleration) Filter(ctx context.Context, state fwk.CycleState, pod
 		return nil
 	}
 
-	errReason := fmt.Sprintf("node(s) had untolerated taint {%s: %s}", taint.Key, taint.Value)
+	klog.V(1).Info("node had untolerated taints", "node", klog.KObj(node), "pod", klog.KObj(pod), "untoleratedTaint", taint)
```

Comment:
This level 1 seems too high? `Filter` is executed for every node for each scheduling cycle. Doesn't it have any perf concern?